src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java

```diff
@@ -177,14 +173,6 @@ public static void registerCustomCodecs(boolean enableFfmpeg)
                 "com.sun.media.codec.video.colorspace.RGBScaler",
                 PlugInManager.CODEC);
 
-        // Remove JMF's H263 codec.
```

Comment:
If JMF has a 263 impl we probably still want to disable it right?

Reply:
We don't use JMF but fmj and I was seraching for those classes and could not find those, that's why I deleted this

Reply:
D'uh of course...sorry for the noise